### ISSUE TITLE
Tests on node 10 fail the installation due to an update of memwatch-next(node-memwatch)

### ISSUE THREAD

**Issue by jannyHou** [strongloop/loopback-connector-mongodb#451]
# Description/Steps to reproduce

The tests running with node 10 fail due to an update of `memwatch-next`.
See issue https://github.com/marcominetti/node-memwatch/issues/39

The fix for a node 10 installation would be updating the dependency from 

```js
// compatible with node 6, 8
"memwatch-next":  "^0.2.0"
```

to 

```js
// compatible with node 10 only
"node-memwatch": "^1.0.0"
```

While that *breaks the installation on nodejs 8*

# Expected result

CI passes for node 6,8,10

# Additional information

To solve the installation error, we could either install the dependency programmatic according to the detected nodejs version, or find a module that replaces `memwatch-next`.

**Comment by virkt25** [comment]
The module is used for leak detection and is never actually run as part of our test suite. The failure comes from just trying to install the dependency on a given version. I explored alternative modules as suggested by @jannyHou but then the same issue happens for Node 6 instead of 10. 

The consensus was to remove the dependency entirely from the package and update the readme with instructions to explain which dependency to install (without saving into `package.json`) including the why it's not there and has to be installed separately. And then just explain running the leak detection command.

**Comment by virkt25** [comment]
fixed by #458
